pkg/fftm/manager.go

```diff
@@ -178,6 +124,13 @@ func (m *manager) initServices(ctx context.Context) (err error) {
 			return err
 		}
 	}
+
+	m.txHandler, err = txhandlerfactory.NewTransactionHandler(ctx, tmconfig.PolicyEngineBaseConfig, config.GetString(tmconfig.PolicyEngineName))
```

Comment:
If we're going for the full rename, shouldn't `PolicyEngineName` change?

Reply:
Updated PR to support old and new configuration spellings